custom_components/tahoma/lock.py

```diff
@@ -46,49 +43,44 @@ def update(self):
             return
 
         self.controller.get_states([self.tahoma_device])
-        self._battery_level = self.tahoma_device.active_states["core:BatteryState"]
-        self._name = self.tahoma_device.active_states["core:NameState"]
-        if (
-            self.tahoma_device.active_states.get("core:LockedUnlockedState")
-            == STATE_LOCKED
-        ):
-            self._lock_status = STATE_LOCKED
-        else:
-            self._lock_status = STATE_UNLOCKED
-        self._available = (
-            self.tahoma_device.active_states.get("core:AvailabilityState")
-            == "available"
-        )
 
     def unlock(self, **kwargs):
         """Unlock method."""
-        _LOGGER.debug("Unlocking %s", self._name)
+        _LOGGER.debug("Unlocking %s", self.name)
         self.apply_action("unlock")
 
     def lock(self, **kwargs):
         """Lock method."""
-        _LOGGER.debug("Locking %s", self._name)
+        _LOGGER.debug("Locking %s", self.name)
         self.apply_action("lock")
 
     @property
     def name(self):
         """Return the name of the lock."""
-        return self._name
+        return self.tahoma_device.active_states["core:NameState"]
 
     @property
     def available(self):
         """Return True if the lock is available."""
-        return self._available
+        return (
+            self.tahoma_device.active_states.get("core:AvailabilityState")
+            == "available"
+        )
 
     @property
     def is_locked(self):
         """Return True if the lock is locked."""
-        return self._lock_status == STATE_LOCKED
+        return (
+            self.tahoma_device.active_states.get("core:LockedUnlockedState")
+            == STATE_LOCKED
+        )
 
     @property
     def device_state_attributes(self):
         """Return the lock state attributes."""
-        attr = {ATTR_BATTERY_LEVEL: self._battery_level}
+        attr = {
```

Comment:
`device_state_attributes` is not necessary, since `ATTR_BATTERY_LEVEL` will be added by `tahoma_device`

Reply:
I prefer to do it in another PR to not increase the size of this one.

Reply:
Also fine, however instead of changing this part, you could just delete the whole `device_state_attributes`.

Reply:
Done.